ui/src/forms/FormProperty.vue

```diff
@@ -26,7 +26,7 @@
 import { Prop, Component, Vue } from 'vue-property-decorator'
 import { PropertyObjectState, PropertyState } from '@hydrofoil/shaperone-core/models/forms'
 import { TemplateResult } from 'lit-element'
-import { dash } from '@tpluscode/rdf-ns-builders'
+import '@rdfine/dash/extensions/sh/PropertyShape'
```

Comment:
Makes the `hidden` property visible to typescript
Gotta fix shaperone so that will not be necessary in next version.

Reply:
I think it would make sense to put this comment in the code